debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/pgproto/PgProtoMessageDecoder.java

```diff
@@ -35,6 +35,11 @@
 
     @Override
     public void processMessage(final ByteBuffer buffer, ReplicationMessageProcessor processor, TypeRegistry typeRegistry) throws SQLException, InterruptedException {
+        if (buffer == null) {
```

Comment:
I'd expect the same change should be applied to other `AbstractMessageDecoder` implementations (wal2json, pgoutput? Or rather, it should be extracted into a method on the abstract base class.

Reply:
Yeah, you're right! For some reasons tests passed on my local machine and I missed it. Will try to fix it

Reply:
It's a matter of (Maven) profiles which decoder is used for the tests. decoderbufs is used by default, see [the description](https://github.com/debezium/debezium#running-tests-of-the-postgres-connector-using-the-wal2json-or-pgoutput-logical-decoding-plug-ins) in the README on how to enable the others. CI runs as a matrix job testing all of them.

Reply:
I see, thank you!